libs/ui-toolkit/src/components/callout/callout.tsx

```diff
@@ -27,15 +66,12 @@ export function Callout({
     'p-16',
     getIntentShadow(intent),
     {
-      flex: !!iconName,
+      flex: !!icon,
```

Comment:
Chore (blocking): This will then need to become `flex: !!iconElement`

Reply:
and changed